src/darwin/Framework/CHIP/MTRDevice.mm

```diff
@@ -453,11 +461,31 @@ - (void)_handleEventReport:(NSArray<NSDictionary<NSString *, id> *> *)eventRepor
     NSDate * oldEstimatedStartTime = _estimatedStartTime;
     for (NSDictionary<NSString *, id> * eventDict in eventReport) {
         // Whenever a StartUp event is received, reset the estimated start time
+        //   New subscription case
+        //     - Starts Uncreachable
+        //     - Receive priming report ReportBegin
+        //     - Optionally receive UpTime attribute - update time and save start time estimate
+        //     - Optionally receive StartUp event
+        //       - Set estimated system time from event receipt time, or saved UpTime estimate if exists
+        //     - ReportEnd handler clears the saved start time estimate based on UpTime
+        //   Disconnected subscription case
+        //     Same as the above
```

Comment:
So this is the client-side resubscribe case, as opposed to the server-side resumption cases listed below as "resumption"?  It would be good to make that clearer.

Reply:
Clarified a bit.